drape/CMakeLists.txt

```diff
@@ -178,6 +178,7 @@ target_link_libraries(${PROJECT_NAME}
   expat
   $<$<BOOL:${PLATFORM_MAC}>:-framework\ OpenGL>
   $<$<BOOL:${PLATFORM_LINUX}>:OpenGL::GL>
+  $<$<BOOL:${PLATFORM_LINUX}>:OpenGL::EGL>
```

Comment:
It means that after merging this PR the build will fail on any Linux without EGL installed/available. That's not good for contributors.

Reply:
And without this there is no linking to EGL: `gl_functions.cpp:260: error: undefined reference to 'eglGetProcAddress'`

Reply:
Oh you mean that this should be conditional? I will look into that.

Reply:
This part is conditional already.

Reply:
It should be conditional on the presence of EGL files, detected by CMake. It requires the installation of mesa packages, right? That also requires updating build documentation. And in reality, it is required only for the desktop target build, e.g. not for the generator_tool.